<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchBox</class>
 <widget class="QWidget" name="searchBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>searchBox</string>
  </property>
  <widget class="QLabel" name="label_Picture">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qt_controls_res.qrc">:/controls_res/Icon01.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_input">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>可以使用*号和问号通配符</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qt_controls_res.qrc"/>
 </resources>
 <connections/>
</ui>
